open-access-variant-filtering-wf/pkg.json

```diff
@@ -16,7 +16,7 @@
     "dependencies": [
         "github.com/icgc-argo/data-processing-utility-tools/cleanup-workdir@1.0.0",
         "github.com/icgc-argo/data-processing-utility-tools/helper-functions@1.0.0",
```

Comment:
would it be good to use this opportunity to update these util packages to use the new github org? it's up to you

Reply:
thanks Junjun. I thought about it. The current versions in new repos are 4 digits version(temporary ones), so I give up to update them. In principle, they are the same.